OpenEdX/Router.swift

```diff
@@ -210,6 +210,26 @@ public class Router: AuthorizationRouter,
         navigationController.pushViewController(controller, animated: true)
     }
     
+    public func showWebDiscoveryDetails(
+        pathID: String,
+        discoveryType: DiscoveryWebviewType,
+        sourceScreen: LogistrationSourceScreen
+    ) {
+        let view = DiscoveryWebview(
+            viewModel: Container.shared.resolve(
+                DiscoveryWebviewModel.self,
+                argument: sourceScreen)!,
```

Comment:
same here

Reply:
I guess it's fine because having the `(` brackets of the argument on the next line breaks the arguments flow.